CodeForAfricaLabs/CitizenReporter.Android
# Capture Video and Images

#### What does this PR do?
Create a Camera2 API app that captures videos and images.
#### Description of Task to be completed?
Allow user to add video media.
- On click of the video button, user is requested to pick a scene then the scene is overlaid on the camera to help the user capture the video
#### How should this be manually tested?
After cloning the repo, run the app on Android studio.
-The app requires Login in to face-book access
-After login in click to the assignments activity, then click on the video button.
-Select any kind of scene, this should fire up the camera application.
-Try clicking on the camera button or the video button, each button should allow you to capture images or videos respectively. You can also capture images while recording videos
